dev-tools/notice/overrides.json

```diff
@@ -16,3 +16,4 @@
 {"name": "kernel.org/pub/linux/libs/security/libcap/cap", "licenceType": "BSD-3-Clause", "note": "dual licensed as GPL-v2 and BSD"}
 {"name": "kernel.org/pub/linux/libs/security/libcap/psx", "licenceType": "BSD-3-Clause", "note": "dual licensed as GPL-v2 and BSD"}
 {"name": "github.com/awslabs/kinesis-aggregation/go", "licenceType": "Apache-2.0", "url": "https://github.com/awslabs/kinesis-aggregation/blob/master/LICENSE.txt"}
+{"name": "github.com/pascaldekloe/goe", "licenceType": "CC0-1.0", "url": "https://github.com/pascaldekloe/goe/blob/master/LICENSE"}
```

Comment:
Even with this line here and a66b19e68a, `make notice` is failing like so:

```
$ make notice
Generating NOTICE
>> go mod tidy
>> go mod download
>> go list -m -json all | go run go.elastic.co/go-licence-detector -includeIndirect -rules dev-tools/notice/rules.json -overrides dev-tools/notice/overrides.json -noticeTemplate dev-tools/notice/NOTICE.txt.tmpl -noticeOut NOTICE.txt -depsOut
2024/04/26 08:23:30 Failed to detect licences: dependency github.com/pascaldekloe/goe uses licence CC0-1.0 which is not allowed by the rules file
exit status 1
Error: exit status 1
make: *** [notice] Error 1
```

@cmacknz Any ideas what might be going wrong?

Reply:
It is because it is missing from the list of allowed licenses. The override list is to force the detected license, the rules list is for legally permitted licenses.

https://github.com/elastic/elastic-agent/blob/5503e4cc8cabc74655a8aa4771d6b3914996368c/dev-tools/notice/rules.json#L1-L19

Beats already has CC0-1.0 so I assume it is safe to add here.

https://github.com/elastic/beats/blob/0c7845445864d9df34897f5856bffb0e0079b3c1/dev-tools/notice/rules.json#L8

```json
{
  "allowlist": [
    "Apache-2.0",
    "BSD",
    "BSD-2-Clause",
    "BSD-2-Clause-FreeBSD",
    "BSD-3-Clause",
    "CC0-1.0",
```

Reply:
> Beats already has CC0-1.0 so I assume it is safe to add here.

Yeah, CC0-1.0 is also in the default list of legally permitted licenses: https://github.com/elastic/go-licence-detector/pull/16